src/ORM/DataObject.php

```diff
@@ -4372,7 +4425,8 @@ public function provideI18nEntities()
         $singularName = $this->singular_name();
         $conjunction = preg_match('/^[aeiou]/i', $singularName ?? '') ? 'An ' : 'A ';
         return [
-            static::class . '.SINGULARNAME' => $this->singular_name(),
+            static::class . '.CLASS_DESCRIPTION' => $this->classDescription(),
+            static::class . '.SINGULARNAME' => $singularName,
```

Comment:
Replacing `$this->singular_name()` with `$singularName` is unrelated tidyup, since that variable is defined above.

Adding `CLASS_DESCRIPTION` here ensures text collector knows how to collect the value for this key.